kfs-ec/src/main/resources/edu/arizona/kfs/module/ec/document/datadictionary/EffortCertificationDocument.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:p="http://www.springframework.org/schema/p" 
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         
+		http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+  <!-- define the routing types -->  
+  <bean id="RoutingType-EffortCertificationDocument-OrganizationHierarchy" parent="RoutingType-EffortCertificationDocument-OrganizationHierarchy-parentBean">  
+  	<property name="routingAttributes">
+  		<list>
+  			<ref bean="RoutingAttribute-financialDocumentTotalAmount"/>
+  			<ref bean="RoutingAttribute-chartOfAccountsCode"/>
+  			<ref bean="RoutingAttribute-organizationCode"/>
+  			<ref bean="RoutingAttribute-overrideCode"/>
+  			<ref bean="RoutingAttribute-fundGroupCode"/>
+  			<ref bean="RoutingAttribute-subFundGroupCode"/>
+  			<ref bean="RoutingAttribute-objectSubTypeCode"/>
+  		</list>
+  	</property>
+  	<property name="documentValuePathGroups">
+  		<list>
+  			<ref bean="DocumentValuePathGroup-EffortCertificationDocument-OrganizationHierarchy-detailLines"/>
+  		</list>
+  	</property>
+  </bean>  
+  
+  <bean id="DocumentCollectionPath-detailLine-chartOrganizationAndOverrideCode" class="org.kuali.rice.krad.datadictionary.DocumentCollectionPath" abstract="true">
+  	<property name="documentValues">
+  		<list>
+  			<value>chartOfAccountsCode</value>
+  			<value>account.organizationCode</value>
+  			<value>overrideCode</value>
+  			<value>account.subFundGroup.fundGroupCode</value>
+  			<value>account.subFundGroupCode</value>
+  			<value>objectCode.financialObjectSubTypeCode</value>
```

Comment:
Looking at the EffortCertificationDetail.java class, I would think this should be:
<value>financialObject.financialObjectSubTypeCode</value>